tests/integration/lxd/test_launcher.py

```diff
@@ -212,6 +212,29 @@ def test_launch_map_user_uid_true(instance_name, tmp_path):
             instance.delete()
 
 
+def test_launch_map_user_uid_true_no_uid(instance_name, tmp_path):
```

Comment:
These two tests are identical:
`test_launch_map_user_uid_true`
`test_launch_map_user_uid_true_no_uid`

Should the first test use the `uid` parameter?

Reply:
Oops, good catch, the tests shouldn't be identical.